pages/pipelines/skipping.md.erb

```diff
@@ -76,3 +76,15 @@ You can choose to always ignore certain branches. Refer to [Branch configuration
 ## Skip builds using conditionals
 
 You can use conditionals to skip builds at both the pipeline and step level. Refer to [Conditionals](/docs/pipelines/conditionals) for more information.
+
+## Skip builds with existing commits
+
+Sometimes don't want to create a new build if one exists with the same commit, regardless of the branch. For example, when [using merge queues in GitHub](/docs/tutorials/github-merge-queue).
```

Comment:
@MelissaKaulfuss, does this make sense? I used the help text shown in the product for the setting.

It would be good to give a bit of context for why someone might want to do this.